libs/core/src/lib/checkbox/checkbox/checkbox.component.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from '@angular/core';
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 import { FdCheckboxValues } from './fd-checkbox-values.interface';
-import { compareObjects } from '../../utils/public_api';
+import { compareObjects, KeyUtil } from '../../utils/public_api';
```

Comment:
Dont use public_api for imports, referene these files.

Reply:
Done